app/components/public/ticket-list.js

```diff
@@ -190,6 +206,19 @@ export default Component.extend(FormMixin, {
               prompt : this.l10n.t('Please enter the promotional Code')
             }
           ]
+        },
+        price: {
+          identifier : 'price',
+          rules      : [
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Invalid number')
+            },
+            {
+              type   : 'integer[1..]',
```

Comment:
Change the minimum value validation to min price,

Reply:
@kushthedude Couldn't get you. Are you referring to this line?

Reply:
Here the validation you are using is just to check for zero price, Change this to Minimum Value of Donation Ticket Price set up?

Reply:
@kushthedude I've added the logic for minimum price here `app/components/public/ticket-list.js`. We needed the validation there for disabling the order button. :)

Reply:
How will the random buyer know, What's the minimum amount to enable the order button? That's why I asked you to include `Validation for Min Value`, Or else you can write a minimum price in the subscript of the Donation Ticket, Whichever method suits you well.


> @kushthedude I've added the logic for minimum price here `app/components/public/ticket-list.js`. We needed the validation there for disabling the order button. :)

Reply:
@kushthedude Added the info circle icon as a pop-up for this.